build-logic/gradle/src/main/kotlin/convention.lifecycle.gradle.kts

```
@@ -0,0 +1,43 @@
+val taskGroup = "Avito android build"
+
+tasks.named<TaskReportTask>("tasks") {
+    displayGroup = taskGroup
+}
+
+tasks.register("assembleAll") {
+    group = taskGroup
+    description = "Compile all code of the '${project.name}' component"
```

```suggestion
    description = "Compile all code of the '${project.path}' component"
```
Do we need to worry about probable duplicates here?

---

In description probably not, but i will use path anyway